frontend/components/StartLink/_index.html.erb

```diff
@@ -0,0 +1 @@
+<a href="<%= src %>" class="StartLink"><%= text %></a>
```

Comment:
Soz... I'm a little late on the review.

The StartLink's text [is always](https://github.com/barnardos/design-system/blob/master/src/components/StartLink/index.js#L8) "Start now".

Shall we pick this up in a follow-up PR?

Reply:
This site uses internationalisation, so we should be passing this text in, even if it will always be the same text we pass in.

I don't think it would make sense for a component to reach out to the translation library, if we did that then we should rewrite all components to use rails helpers.

For example the above would be re-written as:

``` ruby

Reply:
> This site uses internationalisation, so we should be passing this text in, even if it will always be the same text we pass in.

Makes sense.